src/util/std_expr.h

```diff
@@ -2596,10 +2636,12 @@ template<> inline bool can_cast_expr<bitand_exprt>(const exprt &base)
 class shift_exprt:public binary_exprt
 {
 public:
+  DEPRECATED("use shift_exprt(op0, id, op1) instead")
```

Comment:
Wishlist: many of the above deprecation messages have very meaningful argument names, could this be done here and in some of the below as well?

Reply:
ok, done